.github/workflows/pathogen-repo-ci.yaml

```diff
@@ -201,47 +223,33 @@ jobs:
-      # using the same ref that this workflow was called with (e.g. if this
-      # workflow was invoked by the caller workflow with @foo than we invoke
-      # the action with @foo too), but it's not currently possible to figure
-      # out that ref.  See discussion around this (including results of some
-      # investigation I did):
-      #
-      #   - https://github.community/t/reusable-workflows-get-the-ref-inside-the-called-workflow/224109
-      #   - https://github.community/t/ref-head-in-reusable-workflows/203690/92
-      #
-      # Once we can figure out that ref, then we can actions/checkout our
-      # nextstrain/.github repo at that ref as a sidecar path somewhere and
-      # then invoke the setup-nextstrain-cli action using a local file path
-      # instead of a remote owner/repo path.  This separate checkout will be
-      # necessary since the "uses:" key can't be interpolated (${{…}}) with
-      # context vars.
-      #
-      # For now, update the hardcoded ref (e.g. @90af34…) below when you make
-      # future changes to setup-nextstrain-cli.
-      #
-      # [ Update 16 Feb 2024: We solved this for pathogen-repo-build.yaml, but
-      #   because it required a new permission on the GitHub tokens (id-token:
-      #   write) we decided not to update this workflow (yet?) to use the same
-      #   approach. -trs ]
-      #
-      #   -trs, 28 April 2022
-      - name: Setup Nextstrain
-        uses: nextstrain/.github/actions/setup-nextstrain-cli@c1191de9d5e1a30e91d70b0fd1041d97ed1b2496
+      - # Need to run this after the build repo is cloned so that cloning the
+        # build repo does not overwrite the .git dir and remove the extra support files
+        # that we need from nextstrain/.github repo
+        name: Checkout ${{ needs.workflow-context.outputs.repository }} (sha ${{ needs.workflow-context.outputs.sha }})
+        uses: actions/checkout@v4
         with:
+          repository: ${{ needs.workflow-context.outputs.repository }}
+          ref: ${{ needs.workflow-context.outputs.sha }}
+          path: ${{ env.NEXTSTRAIN_GITHUB_DIR }}
+
+      - name: Setup Nextstrain runtime ${{ matrix.runtime }}
+        uses: ./.git/nextstrain/.github/actions/setup-nextstrain-cli
+        with:
+          cli-version: ">=8.3.0"
           runtime: ${{ matrix.runtime }}
 
+      - name: Run ingest
+        uses: ./.git/nextstrain/.github/actions/run-nextstrain-build-step
+        with:
+          step: ingest
 
-      - run: nextstrain build . ${{ inputs.build-args }}
+      - name: Run phylogenetic
+        uses: ./.git/nextstrain/.github/actions/run-nextstrain-build-step
+        with:
+          step: phylogenetic
 
-      - if: always()
-        uses: actions/upload-artifact@v4
+      - name: Run nextclade
+        uses: ./.git/nextstrain/.github/actions/run-nextstrain-build-step
         with:
-          name: ${{ inputs.artifact-name }}-${{ matrix.runtime }}
```

Comment:
`inputs.artifact-name` isn't being respected any more; it needs to be removed from inputs or (ideally, I think) threaded thru with a slightly changed meaning/usage. See also my other note about CI for `pathogen-repo-ci.yaml`.

Reply:
The point of the CI workflow is supposed to be highly opinionated, low configuration, using a standard interface (the defined sub-dirs and the presence of a CI-specific build configuration).

Currently the output artifact is named: `name: ${{ inputs.directory }}-outputs-${{ inputns.runtime }}` — I think if I change that to something like `name: ci-outputs-${{ inputs.directory }}-${{ inputs.runtime }}` that aligns with the overall "no knobs" design.

Reply:
Not quite. The reason for the `artifact-name` input is so that this repo's own CI distinguishes artifacts between multiple invocations of `pathogen-repo-ci.yaml`.

Reply:
At this point I am not sure what change is being requested. 

It sounds like you're wanting a new `archive-name` input for `pathogen-repo-ci` that would need to get passed down into `run-nextstrain-ci-build` — is that correct?

Reply:
@tsibley ☝️

Reply:
There are two invocations of `pathogen-repo-ci.yaml` in this repo's own `ci.yaml`. What I want is for the artifacts produced by those invocations to not conflict/merge with each other.

Reply:
Added in 3b49692